cmd/virt-prerunner/main.go

```diff
@@ -503,10 +510,15 @@ func startDHCPServer(ifaceName string, mac net.HardwareAddr, ipNet *net.IPNet, g
 		"mac":          mac.String(),
 		"ip":           ipNet.IP.String(),
 		"mask":         net.IP(ipNet.Mask).String(),
-		"gateway":      gateway.String(),
+		"routes":       formatAndSortRoutes(routes),
 		"dnsServer":    strings.Join(resolvconf.GetNameservers(rc.Content, types.IPv4), ","),
 		"domainSearch": strings.Join(resolvconf.GetSearchDomains(rc.Content), ","),
 	}
+
+	if !gateway.Equal(nil) {
```

Comment:
Why not `gateway != nil`?

Reply:
`Equal(nil)` can handle nil and empty.

Reply:
So can `len(gateway) != 0`? It's a bit counter-intuitive to do `gateway.Equal(nil)` if gateway can be nil